### PR TITLE
Fix export of centerOfMass

### LINKED ISSUE #92: Center of Mass not exported
Hi,
I have tried the converter with a simple differential drive robot [dd3.urdf](https://github.com/cyberbotics/urdf2webots/files/5579394/dd3.log) and it seems it doesn't add the center of mass giving this output

```
    INFO: dd3 (PROTO) > Solid > Physics: You must also specify the 'centerOfMass' when specifying the 'inertiaMatrix'
    WARNING: dd3 (PROTO) > Solid > Physics: 'centerOfmass' must also be specified when using an inertia matrix.
```
I tried adding an explicit origin tag in the URDF but that didn't help.

Might be off-topic: Adding the centerOfMass manually to each Solid, when I try to visualise it in Webots it shows this error which I am not sure it is related to the above
```
 INFO: DEF dd3 dd3 (PROTO): A Solid with a non-NULL Physics node must be chosen.
```
Thanks!

### PR DESCRIPTION
**Pull request by omichel**
Fixes #92.
The default value of the center of mass is not (0, 0, 0), but an empty array.
Therefore we should export the value of the center of mass if it is (0, 0, 0).

### CODE REVIEW

**Review by StefanFabian — APPROVED** [review]

**Comment by StefanFabian** [comment]
Any reason why this hasn't been merged yet?
We're getting lots of warnings because of this bug.

**Comment by omichel** [comment]
I need a review from an official maintainer to get it merged.

**Review by stefaniapedrazzi — APPROVED** [review]

**Comment by StefanFabian** [comment]
> I need a review from an official maintainer to get it merged.

Yeah, I know. My question was directed at the maintainers. Sorry for the confusion and thanks for the PR!

**Pull request merged**
